perma_web/perma/migrations/0039_auto_20240412_1801.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.11 on 2024-04-12 18:01
+
+from django.db import migrations
+
+FLAG_NAME = "wacz-playback"
+
+def create_wacz_playback_feature_flag(apps, schema_editor):
+    Flag = apps.get_model("waffle", "Flag")
+    flag = Flag(
+        name=FLAG_NAME,
+        testing=True
+    )
+    flag.save()
+
+def delete_wacz_playback_feature_flag(apps, schema_editor):
+    Flag = apps.get_model("waffle", "Flag")
+    flags = Flag.objects.filter(name=FLAG_NAME)
+    flags.delete()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('perma', '0038_auto_20240318_2102'),
```

Comment:
I think this migration needs to mention Waffle too, like this:
https://github.com/harvard-lil/perma/blob/develop/perma_web/perma/migrations/0038_auto_20240318_2102.py#L28

Reply:
Done!